modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

```diff
@@ -0,0 +1,126 @@
+= File Lifecycle Management
+:toc: right
+
+Manage the lifecycle of files within ownCloud to keep your working structure clean, organized and up to date. Choose an archiving and expiration policy that suits your workflow and select an appropriate archive destination to store your old files. Policies can be configured to include and exclude certain files, users
+and groups from the archiving actions. 
+
+== Installation & Configuration
+
+=== Upload Time Tracking
+
+The first part of file lifecycle management begins with knowing when files first enter ownCloud. To start tracking this the app must be enabled. This will store an `upload-time` with each file, which is the server time at which it first appeared on the ownCloud server. Only files are tracked, not folders since these are never archived, only the files within them.
```

Comment:
`Only files are tracked, not folders since these are never archived, only the files within them.` -->
`Only files in folders are tracked. Folders are never archived and therefore not tracked.`

Reply:
What about files without folders? Are they not tracked ? :)